app/helpers/application_helper.rb

```diff
@@ -16,4 +16,19 @@ def add_or_already_in_tribe
       button_to "Add #{@developer.name} to the tribe", tribe_path(developer_id: @developer.id), id: "signup-button"
     end
   end
+
+  def contractor_summary(contractor)
+    content_tag :div, class: "row" do
+      concat(image_tag( contractor.logo , height: "100px", id: "contractor-#{contractor.id}-logo"))
```

Comment:
Line is too long. [99/80]<br>Space found before comma.<br>Space inside parentheses detected.